src/Endatix.Extensions.Hosting/AppBuilderExtensions.cs

```diff
@@ -22,7 +23,13 @@ public static IEndatixMiddleware UseEndatixMiddleware(this WebApplication app)
         app.UseAuthentication()
             .UseAuthorization();
 
-        app.UseSerilogRequestLogging();
+        app.UseSerilogRequestLogging(options =>

```

Comment:
It's not great that we cannot use appSettings for this change, but if that's what it takes I am OK.

One code style suggestion: imo this code is a big hard on the eye. How about separate statements for readability?

```c#
options.GetLevel = (httpContext, elapsed, ex) =>
{
    if (httpContext.Request.Path.StartsWithSegments("/healthz"))
    {
        return LogEventLevel.Verbose;
    }

    return LogEventLevel.Information;
};
```

Reply:
It shouldn't be a problem that it's not configurable through appSettings as this logging is not needed. If due to some reason this is necessary, it can always but turned on from the settings by setting the level temporarily to Verbose.
It doesn't seem hard to read for me as it is a simple use of the ternary conditional operator but I'll change it to if block so it is clearer.